lib/utils/request.js

```diff
@@ -42,8 +42,13 @@ function createErrorMessage(err, httpResponse, body){
     err.requestId = requestId;
     return err;
   } else {
-    var msg = util.format(i18n._('Error - not 2xx status code. (%s)\n'), statusCode);
-
+    var errorMsgTxt = '';
+    if(statusCode === 401 || statusCode === 403){
```

Comment:
403 status code means that user has no permissions. I think for that error message should be different as logging again will not help here :P